webpack/ForemanInventoryUpload/Components/TabHeader/TabHeader.js

```diff
@@ -22,7 +22,7 @@ const TabHeader = ({ exitCode, onRestart, onDownload, toggleFullScreen }) => (
             {__('Generate and upload report')}
           </Button>
         ) : null}
-        {onDownload ? (
+        {onRestart ? (
```

Comment:
This did not work. Nothing is downloaded when clicking the button.

Reply:
I think the `Download button` should gray out until the report is generated.

Reply:
> I think the `Download button` should gray out until the report is generated.

Can we even do that with this old React and Redux code?

Reply:
@jeremylenz Is it doable?

Reply:
PF5 (and PF4) buttons have `isLoading` and `isDisabled` props. It's quite doable! I would recommend changing the import from `patternfly-react` to `@patternfly/react-core` for everything here.

Reply:
The only reason these aren't PF5 already is that the codemod we ran was only for PF4 > PF5.

Reply:
Is there way to specifically use pf4 when doing the imports? Do you see the page breaking when we change the imports.

Reply:
Importing from `@patternfly/react-core` will use PF5. (It would use PF4 if our version of `@patternfly/react-core` were pinned that way, but it's not.) I don't see any reason not to use PF5 here.

Reply:
Sounds good will give that a go and watch the world burn 😂

Reply:
I think some props of `Grid` are changed since PF3, so you may have to adjust those. Other than that I don't foresee any issues :angel: